scripts/build-rhcc-image.sh

```diff
@@ -50,13 +55,20 @@ if [[ -z "$BUILD" ]]; then
     BUILD=1
 fi
 
+# Use new UBI-based Dockerfile for Server 7.x or later, or SGW 3.x or later
+if [[ ${PRODUCT} == "couchbase-server" && ${VERSION} =~ 6.* ]]; then
+    echo "Using old-school Dockerfile.old"
+    DOCKERFILE=Dockerfile.old
+elif [[ ${PRODUCT} == "sync-gateway" && ${VERSION} =~ 2.* ]]; then
```

Comment:
CB 5.x and SGW 1.x would end up using newer Dockerfile.
It is probably not a big deal since older version are not supported.

Reply:
Yeah, I thought that was a reasonable trade-off for not having to spell out 7.x, 8.x, 9.x ...